api/src/ethereum/parent_mmr_root.rs

```diff
@@ -0,0 +1,37 @@
+//! Ethereum MMR API
+use mmr::{Database, build_client};
+use actix_web::{
+    web::{Data, Path, Json},
+    Responder
+};
+use crate::{Result, Error, AppData};
+use serde::Serialize;
+
+/// MMR root result
+#[derive(Clone, Debug, Serialize, PartialEq, Eq)]
+#[serde(untagged)]
+pub enum MMRRootResult {
+    MmrRoot { mmr_root: String },
+    Error { error: String }
+}
+
+/// Get target mmr
+#[allow(clippy::eval_order_dependence)]
+pub async fn handle(block: Path<String>, app_data: Data<AppData>) -> impl Responder {
+    match mmr_root(block, &app_data.mmr_db) {
+        Ok(root) => Json(MMRRootResult::MmrRoot { mmr_root: format!("0x{}", root) }),
+        Err(err) => Json(MMRRootResult::Error { error: err.to_string() })
+    }
+}
+
+fn mmr_root(block: Path<String>, mmr_db: &Database) -> Result<String> {
```

Comment:
Why still call it mmr_root?

Reply:
rename to parent_mmr_root

Reply:
ok, it will be renamed to parent_mmr_root.